### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by mhoult22** [microsoft/azure-load-testing#95]
**Is your feature request related to a problem? Please describe.**
I would like to use a function from the "[Custom JMeter Functions](https://jmeter-plugins.org/wiki/Functions/)" plugin in my Azure Load Test.

**Describe the solution you'd like**
In this [blog](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/azure-load-testing-create-tests-faster-distribute-input-data-and/ba-p/3471215) it says I will need to wait for a new release of Azure Load Testing to be able to use JMeter plugins, but in this [blog](https://docs.microsoft.com/en-us/azure/load-testing/tutorial-identify-bottlenecks-azure-portal#configure-and-create-the-load-test) you have to add some JMeter plugins to your local JMeter in order to run the test.

So are some plugins included already in Azure Load Testing, and for others we will need to wait for a new release?

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

AB#1587795

**Comment by ninallam** [comment]
@mhoult22 we support plugins that are on https://jmeter-plugins.org/. If you'd like to upload custom jars then that feature is coming soon.

**Comment by mhoult22** [comment]
Hi @ninallam, although it is called "Custom JMeter Functions", it is actually a plugin from https://jmeter-plugins.org/ , the link is https://jmeter-plugins.org/wiki/Functions/ .
If I upload the jar along with the JMeter file should it work, or do I need to do something additional?  I tried uploading the jar along with the jmx but it didn't work for me.  Thanks.

**Comment by ninallam** [comment]
For plugins in https://jmeter-plugins.org/ you don't need to upload a jar. It should work out of the box. Is this not working for you?

**Comment by mhoult22** [comment]
No, it doesn't seem to be working for me.

I have a jmx with a UDV value of `${__env(mythread,,1)}`.  I created this simplified jmx just to test this issue.
![image](https://user-images.githubusercontent.com/67146055/183616516-2013e681-fac2-441a-a122-141126e3233a.png)

If on my Windows laptop I have an environment variable for `mythread` and set a value then JMeter uses that value.
![image](https://user-images.githubusercontent.com/67146055/183616211-6f78eb86-cb6d-464a-8c14-e0bfebeb3ec0.png)

If there is no environment variable then JMeter uses the default value of _1_ in this example.  For this to work locally I had to add the plugin [mentioned above](https://jmeter-plugins.org/wiki/Functions/). 

When I run the same jmx in Azure Load Testing the `env` function is not processed at all and I just get `${__env(mythread,,1)}` in my results.
